lib/resources/example_json/r4_examples_appointment.rb

```diff
@@ -86,6 +86,12 @@ module Resources
             'status': 'accepted'
           }
         ],
+        'patientInstruction': "Preparations:\n- boop\n- Arrive 15 minutes early\n- Can't be empty\n- comment" \
+                              "\nDirections:\n- comment\n- Do not drive after appointment. \r\n\r\nConsider " \
+                              "Bringing a Family Member or Friend to the Doctor's Office. If that is not possible, " \
+                              "the Reception desk can call a cab for you.\nPost Appointment Instructions:\n" \
+                              "- Post Appointment\n- Post Order\n- Be Safe while driving back after visit. Be aware " \
+                              "of white walkers.\n- Post Order\n- comment\nPost Appointment Instructions.:\n- Comment",
```

Comment:
Do we need to include this in the video visit read example as well?

Reply:
We did not include this in the video visit feature. The reason why I did not put it here.
https://github.cerner.com/Igneous/tephra_server/blob/master/engines/fhir_appointment_engine/features/read/appointment_system_persona.feature#L77